src/main/java/io/pravega/connectors/flink/FlinkPravegaWriter.java

```diff
@@ -581,9 +581,11 @@ public void restoreState(List<PendingTransaction> pendingTransactionList) throws
 
                 try (
                         EventStreamClientFactory restoreClientFactory = createClientFactory(scope, clientConfig);
-                        TransactionalEventStreamWriter<T> restorePravegaWriter = restoreClientFactory.createTransactionalEventWriter(stream,
-                            eventSerializer,
-                            writerConfig);
+                        TransactionalEventStreamWriter<T> restorePravegaWriter =
+                                restoreClientFactory.createTransactionalEventWriter(name(),
```

Comment:
So just to confirm that THIS is the breaking change you refer to in that previously it did not require a `writerId`.

Reply:
Yes